src/java/org/apache/cassandra/tools/nodetool/CompressionDictionaryCommandGroup.java

```diff
@@ -0,0 +1,295 @@
+            {
+                TableBuilder tableBuilder = new TableBuilder();
+                TabularData tabularData = probe.listCompressionDictionaries(keyspace, table);
+                List<String> indexNames = tabularData.getTabularType().getIndexNames();
+
+                List<String> columns = new ArrayList<>(indexNames);
+                // ignore raw dict
+                columns.remove(CompressionDictionaryDetailsTabularData.TABULAR_DATA_TYPE_RAW_DICTIONARY_INDEX);
+                tableBuilder.add(columns);
+
+                for (Object eachDict : tabularData.keySet())
+                {
+                    final List<?> dictRow = (List<?>) eachDict;
+
+                    List<String> rowValues = new ArrayList<>();
+
+                    for (int i = 0; i < dictRow.size(); i++)
+                    {
+                        if (i == 3) // ignore raw dict
+                            continue;
+
+                        rowValues.add(dictRow.get(i).toString());
+                    }
+                    tableBuilder.add(rowValues);
+                }
+
+                tableBuilder.printTo(probe.output().out);
+            }
+            catch (Exception e)
+            {
+                probe.output().err.printf("Failed to list dictionaries: %s%n", e.getMessage());
+                System.exit(1);
+            }
+        }
+    }
+
+    @Command(name = "export",
+             description = "Export dictionary from Cassandra to local file.")
+    public static class ExportDictionary extends AbstractCommand
+    {
+        @Parameters(index = "0", description = "The keyspace name", arity = "1")
+        private String keyspace;
+
+        @Parameters(index = "1", description = "The table name", arity = "1")
+        private String table;
+
+        @Parameters(index = "2", description = "File name to save dictionary to", arity = "1")
+        private String dictionaryPath;
+
+        @Option(paramLabel = "dictId", names = { "-i", "--id" },
+        description = "The dictionary id. When not specified, the current dictionary is returned.")
+        private long dictId = -1;
+
+        @Override
+        protected void execute(NodeProbe probe)
+        {
+            if (dictId <= 0 && dictId != -1)
+            {
+                probe.output().err.printf("Dictionary id has to be strictly positive number.%n");
+                System.exit(1);
+            }
+
+            try
+            {
+                CompositeData compressionDictionary;
+
+                if (dictId == -1)
+                {
+                    compressionDictionary = probe.getCompressionDictionary(keyspace, table);
+                }
+                else
+                {
+                    compressionDictionary = probe.getCompressionDictionary(keyspace, table, dictId);
+                }
+
+                if (compressionDictionary == null)
+                {
+                    probe.output().err.printf("Dictionary does not exist.%n");
```

Comment:
nit: adding a bit more information in the output
```suggestion
                    probe.output().err.printf("Dictionary%s does not exist for %s.%s.%n", 
                                              dictId == -1 ? "" : " with id " + dictId, 
                                              keyspace, table);
```